examples/values/values.out

```diff
@@ -10,3 +10,5 @@ true
 23
 Ballerina
 true
+
+
```

Comment:
Extra space ?

Reply:
With the [changes to the values example, L45 to L50](https://github.com/ballerina-platform/ballerina-lang/pull/12851/files#diff-0cbeed89f2ba6f6e554af6c820f43d60R45), these two new lines will be printed for the `nil` examples.